nextcloudappstore/core/views.py

```diff
@@ -45,6 +52,24 @@ def get_queryset(self):
             app=app)
 
 
+class AppCommentApi(View):
+
+    def get(self, request, *args, **kwargs):
```

Comment:
I'm unsure if you can just extend a DetailView and reuse more code, just take a look ;)